### ISSUE TITLE
Mass following doesn't work

### ISSUE THREAD

**Issue by scarlet-tobar** [kensanata/trunk#1]
Problem: When someone tries to follow a list, an error happens. (Mojolicious error screen appears)
Log:
[trunk.log](https://github.com/kensanata/trunk/files/4064708/trunk.log)
It seems like it can't go to the autentication mastodon page.

Thank you for reading.

**Comment by kensanata** [comment]
Yeah, this [an unsolved problem](https://alexschroeder.ch/software/Client_authentication_failed). I ended up disabling it on my instance.
